source/frontend-packages.rst

```diff
@@ -40,13 +40,134 @@ You can download and add these from this documentation:
+
+Remove it also from ``INSTALLED_APPS`` and remove ``COMPRESS_PRECOMPILERS``. Check
+your settings for any other place to remove references to it.
+
+You'll need to remove compressor from the ``base.html`` where it is used to
+include both Javascript and Less stylesheets. Remove the ``{% load compressor %}``
+line at the top of the template. We'll change the compressor tags for
+the Javascript first, which should currently look like this::
+
+    {% compress js %}
+        <script src="{% static 'js/site.js' %}"></script>
+    {% endcompress %}
+
+but, this may include more than one script tag. If it does, you'll need to follow
+the section on `Existing Javascript` to migrate them to a bundle. This line will be
+replaced with one tag loading our combined bundle of all project Javascript::
+
+    <script src="{% static 'js/bundle.js' %}"></script>
+
+The section using compressor to compile Less stylesheets will also be replaced, from
+this::
+
+    {% if debug %}
+        <link rel="stylesheet/less" type="text/css" media="all" href="{% static 'less/site.less' %}">
+        <script src="//cdnjs.cloudflare.com/ajax/libs/less.js/1.5.1/less.min.js"></script>
+    {% else %}
+        {% compress css %}
+            <link rel="stylesheet" type="text/less" media="all" href="{% static 'less/site.less' %}">
+        {% endcompress %}
+    {% endif %}
+
+to this::
+
+    <link rel="stylesheet" media="all" href="{% static 'css/bundle.css' %}">
+
+Existing javascript
+'''''''''''''''''''
+
+If you have a frontend-light project that has only a single JS file, rename it to
+``static/js/index.js`` in the project. If you have multiple scripts, you'll need
+to create this file and import the different scripts in your project. (Read the
+section below on `Using Frontend Packages`)
+
+Existing Stylesheets
+''''''''''''''''''''
+
+If you have a single stylesheet you'll need to rename it ``less/index.less`` and if you have
+multiple you'll need to create this single stylesheet and, within it, import other stylesheets
+like
+
+.. code-block:: less
+
+    @import "./states.less";
+    @import "./base.less";
+    @import "./header.less";
+
+Modernizr
+'''''''''
+
+Your project likely includes a copy of Modernizr, but the new frontend setup builds
+a recent copy of this (and is configurable), so we can change this line::
+
+    <script src="//cdnjs.cloudflare.com/ajax/libs/modernizr/2.7.1/modernizr.dev.js"></script>
+
+to this::
+
+    <script src="{% static 'libs/modernizr.js' %}"></script>
+
+jQuery
+''''''
+
+If you need the jQuery library accessible for script tags that might be in specific pages
+and won't be pulled in by the bundling process, then it might be prudent to keep the
+jquery library included by the existing ``<script>`` tag, and that's okay. You might
+keep the need to move this Javascript out of templates and into modules as part of
+improvements to make down the road.
```

Comment:
This last sentence is a little confusing to me. Maybe add 'for now' to the previous sentence. And then change the final sentence to 'You can move this JS out of the template in a future improvement.' or something similar.